apps/aehttp/priv/swagger.yaml

```diff
@@ -3281,6 +3281,8 @@ definitions:
     properties:
       reason:
         type: string
+      error_code:
```

Comment:
can it be just `code`? As far as it is already in context of `Error`
seems I have troubles with snake keys in node's exceptions

Reply:
This can be confusing with contract code, so rather not.